frontend/src/libraries/language.ts

```diff
@@ -56,7 +56,7 @@ function findBestSupportedLanguage(
   const narrowedLanguages = browserLanguages.map((language) => {
     // Handle special cases for traditional Chinesee fallback
     if (["zh-TW", "zh-HK"].includes(language)) {
```

Comment:
Good question, we detect the language based on the `accept-language` header of the request, and it depends on the browser setting for "Preferred language for displaying pages".
Refer to following table

| Preferred language | Locale |
| --- | --- |
| Chinese - Taiwan (zh-TW) | zh-hant |
| Chinese - Hong Kong (zh-HK) | zh-hant |
| Chinese - Singapore (zh-SG | zh-hans |
| Chinese - China (zh-CN) | zh-hans |
| Chinese - (zh) | zh-hans |

"Preferred language" column is the supported languages on popular browsers (i.e. Chrome, Firefox)
And "Locale" column is the official locale.

PS: I  prefer a case insensitive approach for the follow line, but I think it works as is.
`    if (["zh-TW", "zh-HK"].includes(language)) {`

Reply:
yes. zh-TW and zh-HK are set to the locale `zh-hans` because we didn't have `zh-hant` translation at that time. it's the expected behavior to use `zh-hant` for zh-TW and zh-HK.

Reply:
It's quite mysterious for me why did we have this code before as we had only one `zh-` locale and it was matched for any `zh-` preferred language from the browser.

> // Handle special cases for traditional Chinesee fallback

Probably this comment should be different now? Something about `zh-TW` and `zh-HK` preferred languages, I guess.

Reply:
Change the comment to
```ts
// Set zh-Hant (Traditional Chinese) as the preferred languages for zh-TW and zh-HK
```